bin/steps/cryptography

```diff
@@ -20,18 +20,16 @@ source $BIN_DIR/utils
 bpwatch start libffi_install
 
 # If a package using cffi exists within requirements, use vendored libffi.
-if (pip-grep -s requirements.txt bcrypt cffi cryptography django[bcrypt] Django[bcrypt] pyOpenSSL PyOpenSSL requests[security] &> /dev/null) then
+if (pip-grep -s requirements.txt bcrypt cffi cryptography django[bcrypt] Django[bcrypt] PyNaCl pyOpenSSL PyOpenSSL requests[security] misaka &> /dev/null) then
 
-  if [ -d ".heroku/vendor/lib/libffi-3.1.1" ]; then
-    export LIBFFI=$(pwd)/vendor
-  else
+  if [ ! -d ".heroku/vendor/lib/libffi-3.1" ]; then
```

Comment:
Feel free to understand yourself:
https://github.com/Lightricks/heroku-buildpack-python-openssl-1.0.2/commit/5438aab12146a9a1addce263789fd2cd62df1de7